table/sst_file_writer.cc

```diff
@@ -45,6 +45,8 @@ struct SstFileWriter::Rep {
     // TODO (hx235): pass in `WriteOptions` instead of `rate_limiter_priority`
     // during construction
     write_options.rate_limiter_priority = io_priority;
+    ts_sz = _user_comparator->timestamp_size();
+    strip_timestamp = ts_sz > 0 && !ioptions.persist_user_defined_timestamps;
```

Comment:
Minor but these could be done in the member initialization list above (as opposed to the ctor body)